From 4c6f06865721b189f215683aa0e58b8729c0cd54 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Wed, 11 Apr 2018 16:12:44 +0000
Subject: [PATCH] OS-6886 SMB libraries are missing Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 manifest | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/manifest b/manifest
index 24dec079c9..b4028fcaa7 100644
--- a/manifest
+++ b/manifest
@@ -9995,13 +9995,6 @@ d usr/lib/smbfs 0755 root bin
 f usr/lib/smbfs/smbiod 0555 root bin
 f usr/lib/smbfs/smbiod-svc 0555 root bin
 d usr/lib/smbsrv 0755 root bin
-d usr/lib/smbsrv/amd64 0755 root bin
-f usr/lib/smbsrv/amd64/libmlsvc.so.1 0755 root bin
-s usr/lib/smbsrv/amd64/libmlsvc.so=libmlsvc.so.1
-f usr/lib/smbsrv/amd64/libsmb.so.1 0755 root bin
-s usr/lib/smbsrv/amd64/libsmb.so=libsmb.so.1
-f usr/lib/smbsrv/amd64/libsmbns.so.1 0755 root bin
-s usr/lib/smbsrv/amd64/libsmbns.so=libsmbns.so.1
 d usr/lib/smbsrv/dtrace 0755 root bin
 f usr/lib/smbsrv/dtrace/smbd-all.d 0555 root bin
 f usr/lib/smbsrv/dtrace/smbd-authsvc.d 0555 root bin
@@ -10011,6 +10004,7 @@ f usr/lib/smbsrv/dtrace/smbnode.d 0555 root bin
 f usr/lib/smbsrv/dtrace/smbsrv.d 0555 root bin
 f usr/lib/smbsrv/dtrace/smbvfs.d 0555 root bin
 f usr/lib/smbsrv/libmlsvc.so.1 0755 root bin
+f usr/lib/smbsrv/libsmb.so.1 0755 root bin
 f usr/lib/smbsrv/libsmbns.so.1 0755 root bin
 f usr/lib/smbsrv/smbd 0755 root bin
 d usr/lib/smedia 0755 root bin
-- 
2.21.0

